src/main/java/lux/fm/bookingservice/model/Role.java

```diff
@@ -29,4 +29,9 @@ public class Role implements GrantedAuthority {
     public String getAuthority() {
         return name.name();
     }
+
+    public enum RoleName {
```

Comment:
We could further rename it to just Name. So it will be used as Role.Name. Also Nazar said smth about including Role class in the User class directly as it's unique to the User.

Reply:
He said it about Enum